src/native/libs/System.Globalization.Native/pal_icushim.c

```diff
@@ -538,7 +540,7 @@ void GlobalizationNative_InitICUFunctions(void* icuuc, void* icuin, const char*
 #if defined(TARGET_WINDOWS)
         sprintf_s(symbolSuffix, SYMBOL_CUSTOM_SUFFIX_SIZE, "_%s", suffix);
 #else
-        sprintf(symbolSuffix, "_%s", suffix);
+        snprintf(symbolSuffix, SYMBOL_CUSTOM_SUFFIX_SIZE, "_%s", suffix);
```

Comment:
>SYMBOL_CUSTOM_SUFFIX_SIZE [](http://example.com/codeflow?start=31&length=25)

should this be `suffixAllowedSize`?

Reply:
I think we should match the Windows case here as it's effectively the same bounds check.

Reply:
Fine with me if you want to keep it as Windows does it. `SYMBOL_CUSTOM_SUFFIX_SIZE` still safe.